alerts/README.md

```diff
@@ -43,6 +43,10 @@ This plugin extension contains [Prometheus Alertmanager](https://github.com/prom
 | `silenceExcludedLabels` | SilenceExcludedLabels are labels that are initially excluded by default when creating a silence. However, they can be added if necessary when utilizing the advanced options in the silence form.The labels must be an array of strings. Example: `["pod", "pod_name", "instance"]`                                                            | `[]`           |
 | `filterLabels`          | FilterLabels are the labels shown in the filter dropdown, enabling users to filter alerts based on specific criteria. The 'Status' label serves as a default filter, automatically computed from the alert status attribute and will be not overwritten. The labels must be an array of strings. Example: `["app", "cluster", "cluster_type"]` | `[]`           |
 
+| `silenceTemplates` | silenceTemplates are used in the Modal (Schedule Silence) to allow pre-defined silences to be used to scheduled Maintenance Windows. The format consists of a list of objects including description, editable_labels (array of strings specifying the labels that users can modify), fixed_labels (map containing fixed labels and their corresponding values), status, and title. | `[]` |
```

Comment:
- `silenceTemplates are used...`: please use also capital letters
- `Modal (Schedule Silence)`: please in lowercase
- `Maintenance Windows`: lowercase